src/services/__tests__/run-script-manager.test.ts

```
@@ -199,6 +270,92 @@ describe('RunScriptManager', () => {
     });
   });
 
+  describe('spawn error handling', () => {
+    it('should reject when spawn emits error', async () => {
+      mockSpawn.mockImplementation(() => {
+        setTimeout(() => {
+          const errorHandlers = eventHandlers.get('error') || [];
+          errorHandlers.forEach((handler) => handler(new Error('spawn ENOENT')));
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)